examples/.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Hmm, this is strange.
I've modified the file, squashed commits and pushed. 
The file now shows 2 lines in my repo, yet on pull request it still shows as missing newline character.